betterconf/providers.py

```diff
@@ -30,3 +32,12 @@ def __init__(self, files: Union[str, List[str]],
 
     def get(self, name: str) -> str:
         return self.cfg.get(name)
+
+class TOMLProvider(AbstractProvider):
+    """Provider that gets values from .TOML files"""
+
+    def __init__(self, files: Union[str, List[str]]):
+        self.data = toml.load(files)
```

Comment:
I don't think adding extra dependencies is a good idea. I suppose betterconf is meant to be minimalistic and dependency-free library. As a compromise I can suggest add `load` parameter to `TOMLProvider`'s constructor, this way:
```python
def __init__(self, load: Callable[[Union[str, list, IO[str]]], dict], files: Union[str, List[str], IO[str]]):
        self.data = load(files)
```
I added `IO[str]` to types because it is also accepted by `toml.load` ([click](https://github.com/uiri/toml/blob/230f0c9cffd230354759949dd1dd89acd3dddb80/toml.pyi#L18)) .

Reply:
Hm. That looks good. Thanks!